samples/trace/README.md

```diff
@@ -5,6 +5,8 @@ This example shows how to use [@opentelemetry/node](https://github.com/open-tele
 ## Installation
 
 ```sh
+$ # from root of repo, build all packages
+$ npm install
```

Comment:
Why do we need to run this command on the root? I thought samples are always running against published version?

Reply:
Since I changed the sample to use the repo version, it needs to be compiled. Not necessary if it runs against the published version

Reply:
I see, I don't have a strong objection to this.